harness/determined/pytorch/_pytorch_context.py

```diff
@@ -800,6 +803,20 @@ def current_train_batch(self) -> int:
             raise det.errors.InternalException("Training hasn't started.")
         return self._current_batch_idx
 
+    def get_tensorboard_writer(self) -> Any:
+        # Return a managed PyTorch SummaryWriter object
```

Comment:
This is a user-facing object, and so it needs a user-facing docstring.

- What does it return?  (you have that basically covered)
- What benefit do we give to users / why do we need them to call this?